README.md

```diff
@@ -73,6 +74,35 @@ This API is designed to run in 4 environments:
 | Pre-Production | On Merge | Real - copy of live imported every couple of weeks | Internal Testing / QA / External Consumer Testing |
 | Production     | Manually | Real                                               | Live Services                                     |
 
+## Security
+This service is accessed exclusively through the API and has no other user interfaces.
+Onboarding new clients is a manual process and there is no other way to gain access credentials. 
+All in-flight requests are encrypted and sent over HTTPS over the public internet. Any data at rest is encrypted with AWS KMS.
+
+IP Restrictions are in place and will prevent the majority of unauthorised access attempts. We do not currently have AWS WAF applied on our ingress.
```

Comment:
Feel that the security of the Development environment is less interesting to document. This will remain our own volatile playground and carries no risk.

Based on other similar services, it looks like IP restrictions are quite frequently used. Without a firewall, we would get random bad actors from around the world port knocking on our IPs, which is quite loud and fills up error logs.